src/BUILD

```diff
@@ -109,6 +112,7 @@ cc_library(
         "rest_utils.hpp",
         "s3filesystem.cpp",
         "s3filesystem.hpp",
+        "session_id_t.hpp",
```

Comment:
Other headers containing *just* typedefs do not contain `_t` in file name, are you sure this is correct convention?